app/services/users.js

```diff
@@ -24,6 +24,22 @@ exports.createUser = userInfo => {
   return createdUser;
 };
 
+exports.upsertAdmin = async userInfo => {
+  const { email } = userInfo;
+  const userInfoWithRole = { ...userInfo, isAdmin: true, isRegular: false };
+  const existingUser = await userModel.findOne({ where: { email } });
+
+  if (!existingUser) {
+    return exports.createUser(userInfoWithRole);
+  }
+
+  const {
+    1: [{ dataValues: updatedUser }]
```

Comment:
This is so weird, What is the idea?

Reply:
const [_, rowAffected ] = await userModel.update(userInfoWithRole, { where: { email }, returning: true, individualHooks: true });